src/test/java/com/artipie/vertx/VertxSliceServerTest.java

```diff
@@ -193,6 +194,16 @@ public void exceptionInBody() {
         MatcherAssert.assertThat(response, new IsErrorResponse(exception));
     }
 
+    @Test
+    public void serverMayStartOnRandomPort() {
+        final VertxSliceServer srv = new VertxSliceServer(
```

Comment:
@olegmoz How about we add an additional constructor to `VertxSliceServer` that will not have port as an argument and will pass value 0 as the port number to the main constructor. In that way we simplify the usage of that class, the user should not care nor remember that port 0 means a random port.

Reply:
@Vatavuk Totally agree, added constructor without port parameter.